src/sentry/tasks/post_process.py

```diff
@@ -338,32 +362,37 @@ def handle_group_owners(project, group, issue_owners):
 
             # Owners already in the database that we'll keep
             keeping_owners = set()
-            for owner in current_group_owners:
-                owner_type = (
+            for group_owner in current_group_owners:
+                owner_rule_type: str = (
```

Comment:
this annotation is wrong -- you shouldn't need to annotate locals unless they are generic containers and mypy tells you to

Reply:
good catch! this was one of my attempts to fix the typing error here that ended up having to do with variables in loops having the same name, and i had forgotten to take out this attempt 😅